### PR TITLE
Style/grid

### PR DESCRIPTION
**Pull request by aviadamar**
** FRONT**
- Bugs fixing

** BACK **
- Lints bags fixing 
- Changing rout name for calendar view to /calendar
- Changing html file for calendar view to "calendar.html"

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "python.linting.enabled": true
```
Comment (ivarshav) [inline .vscode/settings.json:2]: gitignore

File: .gitignore [hunk 2]
```diff
@@ -9,6 +9,8 @@ __pycache__/
 # Distribution / packaging
```
Comment (ivarshav) [inline .gitignore:9]: I think it will be less confusing to give each PR a unique name :)

File: app/main.py [hunk 3]
```diff
@@ -30,3 +32,22 @@ def profile(request: Request):
         "username": current_username,
         "events": upcouming_events
     })
+
+
+@app.get("/calendar")
```
Comment (ivarshav) [inline app/main.py:37]: Should be in routers/calender.py

File: app/main.py [hunk 4]
```diff
@@ -30,3 +32,22 @@ def profile(request: Request):
...
+            'date': date,
+            'strf_date': routers.calendar_grid.get_date_as_string(date),
+            'days_of_the_week': routers.calendar_grid.DAYS_OF_THE_WEEK,
+            'month_block': routers.calendar_grid.get_month_block(
+                datetime.date(date.year, date.month, 1)
+            )
+        }})
+
+
+if __name__ == "__main__":
```
Comment (ivarshav) [inline app/main.py:52]: please remove

File: app/routers/calendar_grid.py [hunk 5]
```diff
@@ -0,0 +1,49 @@
+import calendar
+import itertools
+from datetime import datetime, timedelta
+
+from typing import Any, Iterator, Generator, List
+
+DAYS_OF_THE_WEEK: List[str] = ['Monday', 'Tuesday', 'Wednesday',
```
Comment (ivarshav) [inline app/routers/calendar_grid.py:7]: see comments in prev PR and please fix before creating a new PR.

**Review by ivarshav — CHANGES_REQUESTED** [review]

File: app/main.py [hunk 6]
```diff
@@ -1,12 +1,14 @@
+import datetime
+
+import uvicorn
```
Comment (yammesicka) [inline app/main.py:3]: Remove this line

File: app/main.py [hunk 7]
```diff
@@ -30,3 +32,22 @@ def profile(request: Request):
         "username": current_username,
         "events": upcouming_events
     })
+
+
+@app.get("/calendar")
+async def monthview(request: Request):
+    date = datetime.date.today()
+    return templates.TemplateResponse("calendar.html", {
+        "request": request,
```
Comment (yammesicka) [inline app/main.py:41]: Why you return the request?

**Review by yammesicka — CHANGES_REQUESTED** [review]

File: app/main.py (hunk 3 above)
Comment (aviadamar) [reply app/main.py]: Done

File: app/main.py (hunk 4 above)
Comment (aviadamar) [reply app/main.py]: Done

File: app/routers/calendar_grid.py (hunk 5 above)
Comment (aviadamar) [reply app/routers/calendar_grid.py]: Done

File: app/main.py (hunk 6 above)
Comment (aviadamar) [reply app/main.py]: done

File: app/main.py (hunk 7 above)
Comment (aviadamar) [reply app/main.py]: According to documentation.

**Pull request closed without merging**